build/opt/pdo/etc/template/enclave.toml

```diff
@@ -33,3 +33,6 @@ https_proxy = '${proxy}'
 
 # spid_api_key is a 32-digit hex string tied to the SPID
 spid_api_key = '${spid_api_key}'
+
+# enclave policy file
+policy_file = 'libpdo-enclave.pol'
```

Comment:
is there a reason to break this out separately? could we just include it in the toml file itself?

Reply:
Sure. I can change this.

Reply:
So, I've worked on this some more and I agree that we can definitely streamline this process a bit. So I'll put a placeholder (as with the `spid_api_key` above) for the enclave policy, remove the explicit/manual CDI enclave policy generation step and combine it with the other configuration generation steps. Then populate the policy in the toml with the enclave policy info in that way when we expand the config in `__tools__/expand-config`. Does this make sense? I want to avoid re-working this a second time, so I want to make sure this is the type of change you're looking for.

Reply:
yes. the main goal is to minimize the amount of configuration complexity. fitting the policy into the existing toml files would be the easiest.